src/Module.php

```diff
@@ -29,9 +29,11 @@ public function getConfig()
      */
     public function onBootstrap(MvcEvent $e)
     {
-        $app            = $e->getTarget();
+        $app = $e->getTarget();
+        /** @psalm-suppress PossiblyNullReference */
```

Comment:
Shouldn't be suppressed: should expand baseline, IMO

Reply:
Fixed.